umm-spec-lib/src/cmr/umm_spec/umm_g/granule.clj

```diff
@@ -60,20 +61,19 @@
       :data-provider-timestamps (umm-g->DataProviderTimestamps umm-g-json)
       :collection-ref coll-ref
       ; :data-granule (xml-elem->DataGranule umm-g-json)
-      ; :access-value (cx/double-at-path umm-g-json [:RestrictionFlag])
       :temporal (umm-g->Temporal umm-g-json)
       ; :orbit-calculated-spatial-domains (ocsd/xml-elem->orbit-calculated-spatial-domains umm-g-json)
       :platform-refs (platform/umm-g-platforms->PlatformRefs (:Platforms umm-g-json))
       :project-refs (project/umm-g-projects->ProjectRefs (:Projects umm-g-json))
+      ; :access-value (cx/double-at-path umm-g-json [:RestrictionFlag])
```

Comment:
I see you and Daniel have decided to move lines around for fun :)

Reply:
See the commit message above: "Reformatted code to minimize future merge conflict.". Then it will be like the PR giphy.